app/assets/stylesheets/mobile.css.scss

```diff
@@ -289,3 +294,14 @@ div.note_wrapper p {
     text-align: right;
 }
 
+.btn {
+  padding: 8px 8px;
+  border-radius: 5px;
+  background-image: none;
+  display: inline-block;
+  white-space: none;
+  border: 1px solid transparent;
+  background-color: #999999;
```

Comment:
Color literals like `#999999` should only be used in variable declarations; they should be referred to via variable everywhere else.